### PR TITLE
Hotfix/fix critical dependencies

### PR DESCRIPTION
**Pull request by darkmavis1980**
### Summary of Changes 📋

- !fix: critical dependencies

### CODE REVIEW

**Review by cfrisoli — DISMISSED** [review]

File: package.json
Comment (stevehealy) [inline package.json]: FYI this version doesn't exist yet.

Comment (darkmavis1980) [reply package.json]: @stevehealy weird, it's picked by npm, so the tag must exists

Comment (stevehealy) [reply package.json]: my bad, I read it as 2.50.0. Yes that IS weird.

Comment (darkmavis1980) [reply package.json]: Should I restore it at `2.40.0`?

**Review by david-mooney — DISMISSED** [review]

File: package.json
Comment (stevehealy) [reply package.json]: @darkmavis1980 sorry, didn't spot your last comment. I think fixing the critical alert has to come ahead of the test metrics. So lets go with 2.5.0 (if that's what npm audit is happy with), and we can test on our end if there are any negative effects.
Do you want to merge that today? Or save it for Monday?

**Review by swatinigam — DISMISSED** [review]

File: package.json
Comment (swatinigam) [reply package.json]: Hey team, if it is okay, may be we can wait until Monday to merge this pr to avoid the possibility of the pipeline breaking for blog since it the one repo that uses allure reporting. If this is not worth waiting, we can merge, see what happens in the pipeline and go from there. @darkmavis1980 what do you think?

**Review by denisolvr — DISMISSED** [review]

**Comment by stevehealy** [comment]
@darkmavis1980 FYI cypress-allure have [updated](https://github.com/Shelex/cypress-allure-plugin/releases/tag/v2.40.1) their dependency.
`v2.40.1`

**Review by denisolvr — APPROVED** [review]

**Review by stevehealy — APPROVED** [review]

**Pull request merged**
